carta/cpp/plugins/cartaview/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name"       : "cartaview",
+    "version"    : "0.1",
+    "type"       : "lib",
+    "description": "adds libraries needed for running carta in casa",
+    "about"      : "Written by Jeff Taylor",
+    "depends"    : []
+}
```

Comment:
Okay, whatever would work best would be all right with me. I actually never intended to commit this plugin at this time as I haven't managed to get it to work properly yet. I had the CASA/CARTA combination working in 2 out of 3 environments without it, so I was hoping that it would end up not being necessary, but unfortunately that number has now dropped to 1 out of 3, so something definitely needs to be done; I just need to figure out what.